### ISSUE TITLE
install texlive instead of tinytex? (was: install tinytex instead of texlive)

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/bioconductor#7]
https://github.com/waldronlab/bioconductor_devel/blob/5cade6cbdfce10dd2a5010f91ca25882be1c2780/Dockerfile#L12

See advice at https://github.com/Bioconductor/bioconductor_full/issues/10. This may help reduce the size of the image while still supporting building documentation and checking packages.

**Comment by LiNk-NY** [comment]
I've added this to the `Dockerfile`

https://github.com/waldronlab/bioconductor_devel/blob/master/Dockerfile#L13

**Comment by lwaldron** [comment]
`R CMD check` does not seem to work with tinytex, e.g., error below on `R CMD check HGNChelper`. Is tinytex only intended for running check from within R, not command-line use?

```
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'Rd2.tex' failed.
Messages:
sh: 1: /usr/bin/texi2dvi: not found
Output:

Error in running tools::texi2pdf()
```

**Comment by lwaldron** [comment]
tinytex required some extra tricks for system-wide installation, which you can see commented out in the Dockerfile now. But didn't seem worth the space saving when it kept missing packages I needed to `R CMD check` a specific package, I opted for a standard texlive installation instead. Let me know if the extra GB of size is a hassle and I could go back to the tinytex + extras, but for its intended convenient local use, I don't see much downside to the extra size of a full texlive installation. Closed by commit ed18d62.
